pkg/util/chunk/column.go

```diff
@@ -245,6 +248,54 @@ func (c *Column) appendNullBitmap(notNull bool) {
 	}
 }
 
+// Reserve allocates some memory for the column
+func (c *Column) Reserve(nullBitmapExpectedCap int64, dataExpectedCap int64, offsetExpectedCap int64) {
```

Comment:
It looks to me all the arguments are the extra needed cap, `XXXExpectedCap` is not a good name in this case.

Reply:
renamed